tt_metal/llrt/rtoptions.hpp

```diff
@@ -215,6 +216,7 @@ class RunTimeOptions {
 
     inline bool get_profiler_enabled() { return profiler_enabled; }
     inline bool get_profiler_do_dispatch_cores() { return profile_dispatch_cores; }
+    inline bool get_profiler_sync_enabled() { return profiler_sync_enabled; }
```

Comment:
these functions should all be marked with `const`

Reply:
These are return bools by value, does const mean much here?

Reply:
wow this class is a lot bigger than I thought; maybe it isn't worthwhile to stay consistent with rest of class
usually you should mark member functions const wherever you aren't modifying member variables